Natasha/quiz.txt

```diff
@@ -7,10 +7,10 @@ This file tests your knowledge of Git/GitHub and also demonstrates how to edit a
 Questions
 =========
 
-a) Who is the author of Git?
+a) Who is the author of Git?Linus
 
-c) Git requires internet access to use. True or False.
+c) Git requires internet access to use. True or False.True
```

Comment:
This is false. Git can be used on your personal computer and does not required internet.